src/main/java/codeenthusiast/TrainingCenterApp/muscle/MuscleDTO.java

```diff
@@ -15,6 +15,7 @@
 
     private String name;
     private String description;
+    private String imageUrl;
     private List<Image> images;
```

Comment:
Muscle entity could have multiple images. for example -> front and back view.
Change string imageUrl to List<String> imageUrls and delete List<Image>
Appropiate mapping in MuscleMapper will be necessary also

Reply:
I don't understand when I delete **List<Image> images;** Muscle.class when breaks.

Reply:
breaks ?